django_netjsongraph/utils.py

```diff
@@ -15,7 +15,7 @@ def print_info(message):  # pragma no cover
     print info message if calling from management command ``update_all``
     """
     if 'update_topology' in sys.argv:
-        print('{0}\n'.format(message))
+        print(('{0}\n'.format(message)))
```

Comment:
I don't think we need this change, do we? 
Are you using some software to change python 2 code to python 3? (Which is allowed, don't worry, I only want to know what prompted this!)